src/main/java/de/hpi/bpt/argos/storage/DatabaseAccess.java

```diff
@@ -0,0 +1,101 @@
+package de.hpi.bpt.argos.storage;
+
+import de.hpi.bpt.argos.properties.PropertyEditorImpl;
+import de.hpi.bpt.argos.storage.dataModel.PersistenceArtifact;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+/**
+ * This interface represents the class to access the database (read/write).
+ */
+public interface DatabaseAccess {
+
+	String DATABASE_CONNECTION_HOST_PROPERTY_KEY = "databaseConnectionHost";
+	String DATABASE_CONNECTION_USERNAME_PROPERTY_KEY = "databaseConnectionUsername";
+	String DATABASE_CONNECTION_PASSWORD_PROPERTY_KEY = "databaseConnectionPassword";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2068)